### DISCUSSION TITLE
Push Notification feature

### DISCUSSION THREAD

**Discussion by Python700** [MKT-Playbook/Hub#37] [Feature requests]
Hello, I have a feature for the app. 
My idea is to have the app push notification to users once your data has updated for the new Tour, and when the next Rank is live. Thanks for reading!

**Comment by TitusPosiadly** [comment]
Hey, and thanks for the suggestion. I think push notifications are one of the features I wanted to add from the beginning but never was able to. Usually, you need to run your own server that can send requests to Apple for this to work, which is expensive and a lot of work, but I recently found a service that might overcome these hurdles. I will have to do some more research but this might be something coming to the app in the near future.
